frontend/src/metabase/entities/actions/actions.ts

```diff
@@ -176,6 +179,9 @@ const enableImplicitActionsForModel =
     dispatch(Actions.actions.invalidateLists());
   };
 
+const CREATE_PUBLIC_LINK = "metabase/entities/action/CREATE_PUBLIC_LINK";
+const DELETE_PUBLIC_LINK = "metabase/entities/action/DELETE_PUBLIC_LINK";
+
```

Comment:
@WiNloSt entity actions should use plural naming, i.e. `/actions/`, not `/action/`

Reply:
Dang it! Why do I miss this over and over again.

<img src="https://media0.giphy.com/media/3ohs7KViF6rA4aan5u/giphy.gif?cid=ecf05e47l8z8zhjyz60s9p5fbly3yi363hcb0bh9a6p04ovl&rid=giphy.gif&ct=g" />